recipes/LJSpeech/TTS/vocoder/hifi_gan/hparams/train.yaml

```diff
@@ -0,0 +1,179 @@
+###################################
+# Experiment Parameters and setup #
+###################################
+seed: 1234
+__set_seed: !apply:torch.manual_seed [!ref <seed>]
+output_folder: !ref ./results/hifi_gan/<seed>
+save_folder: !ref <output_folder>/save
+train_log: !ref <output_folder>/train_log.txt
+epochs: 100
+keep_checkpoint_interval: 10
+
+#################################
+# Data files and pre-processing #
+#################################
+data_folder: !PLACEHOLDER # e.g, /datasets/ljspeech
+train_csv: !ref <save_folder>/train.csv
```

Comment:
If using CSV, training on phonemes will be more challenging - phoneme labels consist of more than one character.